app/main.py

```diff
@@ -1,34 +1,95 @@
+from __future__ import annotations
```

Comment:
The `from __future__ import annotations` statement is unnecessary here unless you are targeting a Python version prior to 3.7. If you are using Python 3.7 or later, you can remove this line.